Stratis.Bitcoin/Stratis.Bitcoin.csproj

```diff
@@ -21,7 +21,8 @@
   <ItemGroup>
     <PackageReference Include="DBreeze" Version="1.85.0" />
     <PackageReference Include="Microsoft.Extensions.Logging.Debug" Version="1.1.2" />
-    <PackageReference Include="Serilog.Extensions.Logging.File" Version="1.0.1" />
+    <PackageReference Include="NLog" Version="5.0.0-beta09" />
```

Comment:
Sadly, not possible in this case. NLog is waiting with stable release for .NET Core 2.0. NLog 4 is not .NET Core compatible. Can't fix this without getting rid of NLog ...